.cirrus.yml

```diff
@@ -0,0 +1,23 @@
+smoke_task:
+    env:
+       CIRRUS_CLONE_DEPTH: 10
+    setup_script:
+       - dnf -y install python3 git xz tcpdump nc iproute iputils gcc python3-devel qemu-kvm qemu-img
+       - python3 -m pip install $AVOCADO_SOURCE
+       - python3 -m pip install -r requirements.txt
+       - python3 setup.py develop --user
+    bootstrap_script:
+       - python3 -m avocado vt-bootstrap --vt-skip-verify-download-assets --yes-to-all
+    list_script:
+       - python3 -m avocado list
+    dry_run_script:
+       - python3 -m avocado --show all run --dry-run -- boot
+    container:
+        matrix:
+          - image: fedora:30
+          - image: fedora:29
+    env:
+        matrix:
+          - AVOCADO_SOURCE: avocado-framework==69.1
+          - AVOCADO_SOURCE: avocado-framework==70.0
```

Comment:
The same here, I'd suggest latest and latest LTS (where `avocado-framework<70` should make sure to always use the latest LTS)

Reply:
Good point on the latest LTS (I mean,  < 70.0 is a better specification and avoids updating to 69.2, etc).

Reply:
I'm dropping the latest, non-LTS, release here.

Reply:
thanks